bftengine/src/preprocessor/messages/PreProcessResultMsg.cpp

```diff
@@ -55,7 +55,8 @@ PreProcessResultMsg::PreProcessResultMsg(ClientRequestMsgHeader* body) : ClientR
 std::pair<char*, uint32_t> PreProcessResultMsg::getResultSignaturesBuf() { return getExtraBufPtr(); }
 
 std::optional<std::string> PreProcessResultMsg::validatePreProcessResultSignatures(ReplicaId myReplicaId,
-                                                                                   int16_t fVal) {
+                                                                                   int16_t fVal,
```

Comment:
We decided at the first stage not to touch this message. Please revert all related changes.